go.mod

```diff
@@ -72,7 +72,7 @@ require (
 	github.com/cucumber/messages-go/v16 v16.0.1 // indirect
 	github.com/davecgh/go-spew v1.1.1 // indirect
 	github.com/dgrijalva/jwt-go v3.2.0+incompatible // indirect
-	github.com/docker/distribution v2.7.1+incompatible // indirect
+	github.com/docker/distribution v2.8.1+incompatible // indirect
```

Comment:
We could consider https://github.com/distribution/distribution in the future as github.com/docker/distribution redirects to it.

Reply:
After reviewing it, I don't think we can't change that:

It is an indirect dependency, so the dependencies that use that dependency should change it to be propagated to our go mod/sum files

Reply:
Specifically:
`github.com/docker/go-healthcheck`

Reply:
Ah thanks for checking. I thought we could be able to re-use it somehow.
You reckon that we are good to merge this at it's state? 

> Specifically: `github.com/docker/go-healthcheck`
> 
> and it is marked as readonly so it is unlikely to change

Also this package is archived, I am wondering if there are better alternatives out there.

Reply:
>  You reckon that we are good to merge this at it's state?

Let's merge it